lib/inngest/utils/preflight.ts

```diff
@@ -0,0 +1,48 @@
+// lib/inngest/utils/preflight.ts
+import { failure, type Result, success } from "./results"
+
+export type ProbeResult = {
+	url: string
+	contentType?: string
+	contentLength?: number
+	suspectedAudio: boolean
+}
+
+export async function preflightProbe(url: string, timeoutMs = 6000): Promise<Result<ProbeResult>> {
+	try {
+		const controller = new AbortController()
+		const _t = setTimeout(() => controller.abort(), timeoutMs)
+		try {
+			const head = await fetch(url, { method: "HEAD", signal: controller.signal })
+			const ct = head.headers.get("content-type") || undefined
+			const cl = Number(head.headers.get("content-length") || "0") || undefined
+			if (ct?.startsWith("audio/") && (cl === undefined || cl > 0)) {
+				return success({ url, contentType: ct, contentLength: cl, suspectedAudio: true })
+			}
+		} catch {}
```

Comment:
@cursoragent  Empty catch block silently ignores errors from the HEAD request. Consider logging the error or adding a comment explaining why it's safe to ignore.


```suggestion
		} catch (err) {
		console.warn(`HEAD request failed for ${url}:`,err);
		}
```